web/client/utils/mapinfo/wfs.js

```diff
@@ -93,9 +94,21 @@ const getIdentifyGeometry = point => {
 
 export default {
     buildRequest,
-    getIdentifyFlow: (layer, baseURL, defaultParams) => {
-        const { point, features, ...baseParams} = defaultParams;
+    getIdentifyFlow: (layer = {}, baseURL, defaultParams) => {
+        const { point, features, ...baseParams } = defaultParams || {};
         if (features) {
+            if (baseURL && baseURL !== 'client') {
```

Comment:
Now that `client` has a particular reserved meaing, I'd extract it in a constant and proper document it, so it is clear what is the scope.